torch/ao/ns/fx/n_shadows_utils.py

```diff
@@ -566,9 +568,19 @@ def handle_subgraph_candidate(
             .set_non_traceable_module_classes([OutputLogger, OutputComparisonLogger])
 
         # add a call to prepare_fx on the wrapper module
-        orig_mod_copy_wrapped = torch.ao.quantization.quantize_fx.prepare_fx(
-            orig_mod_copy_wrapped, qconfig_mapping, example_inputs=example_inputs,
-            prepare_custom_config=prepare_custom_config)
+        if custom_prepare_fn is None:
+            orig_mod_copy_wrapped = torch.ao.quantization.quantize_fx.prepare_fx(
+                orig_mod_copy_wrapped, qconfig_mapping, example_inputs=example_inputs,
+                prepare_custom_config=prepare_custom_config)
+        else:
+            if custom_prepare_kwargs is None:
+                custom_prepare_kwargs = {}
+            custom_prepare_kwargs["example_inputs"] = example_inputs
```

Comment:
should we check that the original `custom_prepare_kwargs` don't specify `example_inputs`, `prepare_custom_config`, `qconfig_mapping`?

Reply:
yeah, that makes sense